about/css/style.css

```diff
@@ -0,0 +1,21 @@
+/* top */
+
+.top-wrapper{
+    padding-top: 215px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.top-logo{
+    margin-bottom: 30px;
+}
+
+.top-content{
+    font-family: 'Libre Baskerville';

```

Comment:
I declared common family to all using font-family and add links to Google-Fonts.

Reply:
👍

also, please, take a look into `@font-face` css rule to add custom fonts in project. It can be useful in case we can't or don't want to add `<link ... />` to resources in every `.html` file.